keyboards/kbdfans/d60b/keyboard.json

```diff
@@ -0,0 +1,230 @@
+            {"matrix": [1, 8], "x": 128, "y": 16, "flags": 4},
+            {"matrix": [1, 9], "x": 144, "y": 16, "flags": 4},
+            {"matrix": [1, 10], "x": 160, "y": 16, "flags": 4},
+            {"matrix": [1, 11], "x": 176, "y": 16, "flags": 4},
+            {"matrix": [1, 12], "x": 192, "y": 16, "flags": 4},
+            {"matrix": [1, 13], "x": 208, "y": 16, "flags": 4},
+            {"matrix": [1, 14], "x": 224, "y": 16, "flags": 4},
+            {"matrix": [2, 0], "x": 0, "y": 32, "flags": 4},
+            {"matrix": [2, 2], "x": 32, "y": 32, "flags": 4},
+            {"matrix": [2, 3], "x": 48, "y": 32, "flags": 4},
+            {"matrix": [2, 4], "x": 64, "y": 32, "flags": 4},
+            {"matrix": [2, 5], "x": 80, "y": 32, "flags": 4},
+            {"matrix": [2, 6], "x": 96, "y": 32, "flags": 4},
+            {"matrix": [2, 7], "x": 112, "y": 32, "flags": 4},
+            {"matrix": [2, 8], "x": 128, "y": 32, "flags": 4},
+            {"matrix": [2, 9], "x": 144, "y": 32, "flags": 4},
+            {"matrix": [2, 10], "x": 160, "y": 32, "flags": 4},
+            {"matrix": [2, 11], "x": 176, "y": 32, "flags": 4},
+            {"matrix": [2, 12], "x": 192, "y": 32, "flags": 4},
+            {"matrix": [2, 14], "x": 224, "y": 32, "flags": 4},
+            {"matrix": [3, 1], "x": 16, "y": 48, "flags": 4},
+            {"matrix": [3, 2], "x": 32, "y": 48, "flags": 4},
+            {"matrix": [3, 3], "x": 48, "y": 48, "flags": 4},
+            {"matrix": [3, 4], "x": 64, "y": 48, "flags": 4},
+            {"matrix": [3, 5], "x": 80, "y": 48, "flags": 4},
+            {"matrix": [3, 6], "x": 96, "y": 48, "flags": 4},
+            {"matrix": [3, 7], "x": 112, "y": 48, "flags": 4},
+            {"matrix": [3, 8], "x": 128, "y": 48, "flags": 4},
+            {"matrix": [3, 9], "x": 144, "y": 48, "flags": 4},
+            {"matrix": [3, 10], "x": 160, "y": 48, "flags": 4},
+            {"matrix": [3, 11], "x": 176, "y": 48, "flags": 4},
+            {"matrix": [3, 13], "x": 208, "y": 48, "flags": 4},
+            {"matrix": [3, 14], "x": 224, "y": 48, "flags": 4},
+            {"matrix": [4, 0], "x": 0, "y": 64, "flags": 4},
+            {"matrix": [4, 2], "x": 32, "y": 64, "flags": 4},
+            {"matrix": [4, 3], "x": 48, "y": 64, "flags": 4},
+            {"matrix": [4, 7], "x": 112, "y": 64, "flags": 4},
+            {"matrix": [4, 11], "x": 176, "y": 64, "flags": 4},
+            {"matrix": [4, 13], "x": 208, "y": 64, "flags": 4},
+            {"matrix": [4, 14], "x": 224, "y": 64, "flags": 4}
+        ],
+        "max_brightness": 200,
+        "sat_steps": 8,
+        "sleep": true,
+        "spleep": false,
+        "val_steps": 8
+    },
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x1600",
+        "vid": "0xCA04"
+    },
+    "ws2812": {
+        "pin": "F7"
+    },
+    "layout_aliases": {
+        "LAYOUT_60_tsangn_hhkb": "LAYOUT_60_ansi_tsangan_split_bs_rshift"
+    },
```

Comment:
```suggestion
```

Reply:
As for a previous PR, why (besides the typo that I will correct) can't we keep the alias LAYOUT_60_tsangan_hhkb, is it not a common name for the split bs and rshift ?

Reply:
1. `60_tsangan_hhkb` is the previous name, i.e. no longer used, for community layout `60_ansi_tsangan_split_bs_rshift`
2. Purpose of `layout_aliases` is for backwards compatibility when layout changes are made to an existing keyboard. As this PR is a keyboard addition rather than change to an existing keyboard, `layout_aliases` isn't required.